flake8_trio/visitors/visitor91x.py

```diff
@@ -90,9 +91,125 @@ def checkpoint_statement(library: str) -> cst.SimpleStatementLine:
     )
 
 
+class CommonVisitors(cst.CSTTransformer, ABC):
```

Comment:
The subclassing needed to be expanded to move some import logic into it, since otherwise a file only with yields that can't be autofixed, inside loop bodies, (i.e. tests/eval_files/trio91x_noautofix) had no way of figuring out whether to add an import or not